_layouts/home.html

```diff
@@ -12,14 +12,72 @@ <h2 id="countdown">&nbsp;</h2>
   </div>
+  
+  
+<div id="bootstrap-touch-slider" class="carousel bs-slider fade  control-round indicators-line" data-ride="carousel" data-pause="hover" data-interval="5000" >
 
-  <div class="flex row-xs-center ">
-    <div class="btn-group-lg">
-      <a class="btn btn-lg btn-default btn-next btn-xs-full" href="{{ '/host/' | prepend: site.baseurl }}">Become a host&nbsp;&nbsp;<i class="fa fa-play" aria-hidden="true"></i></a>
-      <a class="btn btn-lg btn-default btn-next btn-xs-full" href="{{ '/participant/' | prepend: site.baseurl }}">Attend an event&nbsp;&nbsp;<i class="fa fa-play" aria-hidden="true"></i></a>
-    </div>
-  </div>
+            <ol class="carousel-indicators">
+                <li data-target="#bootstrap-touch-slider" data-slide-to="0" class="active"></li>
+                <li data-target="#bootstrap-touch-slider" data-slide-to="1"></li>
+                <li data-target="#bootstrap-touch-slider" data-slide-to="2"></li>
+            </ol>
+
+            <div class="carousel-inner" role="listbox">
+
+                <div class="item active">
+
+                    <img src="../img/homepage_slide#1.jpg"/>
+                    <div class="bs-slider-overlay"></div>
+
+                    <div class="container">
+                        <div class="row">
+                            <div class="slide-text slide_style_left">
+                                <p data-animation="animated fadeInLeft">The best is yet to come!!!</p>
+                                <a href="{{ '/participant/' | prepend: site.baseurl }}” target="_blank" class="btn btn-default" data-animation="animated fadeInUp">Become an host</a>
+                                <a href="{{ '/host/' | prepend: site.baseurl }}" target="_blank"  class="btn btn-primary" data-animation="animated fadeInDown">Attend an event</a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="item">
+
+                    <img src="../img/homepage_slide#2.jpg" alt="Bootstrap Touch Slider"  class="slide-image"/>
+                    <div class="bs-slider-overlay"></div>
+                    <div class="slide-text slide_style_center">
+                        <h1 data-animation="animated zoomInRight">Looking forward to meet you in our conference!!!</h1>
+                        <p data-animation="animated fadeInLeft">The best is yet to come!!!</p>
+                        <a href="{{ '/participant/' | prepend: site.baseurl }}” target="_blank" class="btn btn-default" data-animation="animated fadeInUp">Become an host</a>
+                        <a href="{{ '/host/' | prepend: site.baseurl }}" target="_blank"  class="btn btn-primary" data-animation="animated fadeInDown">Attend an event</a>
+                    </div>
+                </div>
+
+                <div class="item">
+
+                    <img src="../img/homepage_slide#3.jpg" alt="Bootstrap Touch Slider"  class="slide-image"/>
```

Comment:
I see this is still linking to images with #. Did you already have the chance to update that part? Also regarding licensing, we will need to give the correct attribution according to the CC License, even if those images will only be temporary.